lib/release_notes_bot_web/controllers/webhook_controller.ex

```diff
@@ -74,13 +74,18 @@ defmodule ReleaseNotesBotWeb.WebhookController do
     persistence_location = determine_persistence(body, project.project_provider)
     project_provider = project.project_provider |> Enum.take(1) |> List.first()
 
-    # TO DO: Handle case where persistence fails (slug = nil)
-    WebhookEventPersistences.create(%{
-      :slug => persistence_location |> String.split("/") |> Enum.take(-2) |> List.first(),
-      :webhook_event_id => ReleaseTags.get_id(repo_id, Integer.to_string(release["id"])),
-      :persistence_provider_id => project_provider.persistence_provider_id,
-      :version => 1
-    })
+    case persistence_location do
+      persistence_location when is_binary(persistence_location) ->
```

Comment:
can this be simplified somehow?

Reply:
Yes, this could be simplified to an if statement, but I have realized that the logic for [MRNN76](https://mojotech.atlassian.net/jira/software/projects/MRNN/boards/22?selectedIssue=MRNN-76) fits right into that catchall case. This case would be triggered when a persistence to Confluence fails. I have added a comment for this case. 76 is blocked by another larger ticket, else I would implement 76 in this PR.

Reply:
I have added this case statement now so that if persistence to confluence does fail, the app will not crash